02-Advanced-topics/04-Rendering-images-in-matplotlib.ipynb

```diff
@@ -0,0 +1,192 @@
+{
```

Comment:
<p>`return np.asarray(out.buf)` should be `return out[:]`. `buf` should not be accessed directly by users.</p><p><br></p>

 _Reply via <a href='https://app.reviewnb.com/glotzerlab/fresnel-examples/pull/4/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>